### PR TITLE
Add ::paginate and fix ::find_by

### LINKED ISSUE #49: find_by not working with multiple fields, only uses the field that was defined first
Relating to this line in README.md:
`Foo.find_by(someField: [searchquery1, searchquery2], someOtherField: "bar").load`

Querying against multiple fields only uses the field that was defined first:
```
[56] pry(main)> ContentfulMeal.find_by(day:2, diet_type:"Gluteeniton").load.map{|cm| cm.diet_type}
=> ["Gluteeniton",
 "Normaali",
 "Normaali",
 "Normaali",
 "Gluteeniton",
 "Normaali",
 "Normaali",
 "Normaali",
 "Gluteeniton",
 "Gluteeniton",
 "Gluteeniton",
 "Gluteeniton",
 "Gluteeniton",
 "Normaali",
 "Normaali",
 "Normaali",
 "Normaali",
 "Gluteeniton",
 "Normaali",
 "Gluteeniton",
 "Normaali",
 "Gluteeniton",
 "Gluteeniton",
 "Gluteeniton",
 "Gluteeniton",
 "Normaali",
 "Gluteeniton",
 "Gluteeniton",
 "Normaali",
 "Normaali",
 "Gluteeniton",
 "Normaali",
 "Gluteeniton",
 "Gluteeniton",
 "Normaali",
 "Normaali",
 "Normaali",
 "Gluteeniton",
 "Normaali",
 "Gluteeniton",
 "Normaali",
 "Gluteeniton"]
```
Above it's returning all meals for day 2, when ir should only return gluten free meals.
Workaround is to chain the `find_by` queries, which isn't as pretty, but works:
```
[57] pry(main)> ContentfulMeal.find_by(day:2).find_by(diet_type: "Gluteeniton").load.map{|cm| cm.diet_type}
=> ["Gluteeniton",
 "Gluteeniton",
 "Gluteeniton",
 "Gluteeniton",
 "Gluteeniton",
 "Gluteeniton",
 "Gluteeniton",
 "Gluteeniton",
 "Gluteeniton",
 "Gluteeniton",
 "Gluteeniton",
 "Gluteeniton",
 "Gluteeniton",
 "Gluteeniton",
 "Gluteeniton",
 "Gluteeniton",
 "Gluteeniton",
 "Gluteeniton",
 "Gluteeniton",
 "Gluteeniton",
 "Gluteeniton"]
[58] pry(main)>
```
Using `contentful_model (0.2.0)`

### PR DESCRIPTION
**Pull request by dlitvakb**
Fixes #79 
Fixes #49

### CODE REVIEW

File: lib/contentful_model/chainable_queries.rb [hunk 1]
```diff
@@ -39,6 +38,14 @@ def locale(locale_code)
         self
       end
 
+      def paginate(page = 1, per_page = 100)
+        page = 1 if page.nil?
+        per_page = 25 if per_page.nil?
+        skip_records_count = (page - 1) * per_page
+        @query << { 'limit' => per_page, 'skip' => skip_records_count }
```
Comment (mariobodemann) [inline lib/contentful_model/chainable_queries.rb]: Do you think you should add the `order` parameter, to "_keep the ordering predictable_"?

https://www.contentful.com/developers/docs/references/content-delivery-api/#/introduction/collection-resources-and-pagination

File: spec/chainable_queries_spec.rb [hunk 2]
```diff
@@ -129,6 +129,38 @@ def invalid?
...
+
+      it 'can change page_size' do
+        expect(subject.paginate(1, 20)).to eq subject
+        expect(subject.query.parameters).to include('limit' => 20, 'skip' => 0)
+      end
+
+      it 'can change page_size and select a different page' do
+        expect(subject.paginate(3, 20)).to eq subject
+        expect(subject.query.parameters).to include('limit' => 20, 'skip' => 40)
+      end
```
Comment (mariobodemann) [inline spec/chainable_queries_spec.rb]: Do you mind checking the outliers too? Like `.paginate(-1,0)` or `.paginate('three')`, and so on?

Comment (dlitvakb) [reply spec/chainable_queries_spec.rb]: 👌 will add.

File: lib/contentful_model/chainable_queries.rb (hunk 1 above)
Comment (dlitvakb) [reply lib/contentful_model/chainable_queries.rb]: Yes, will do

**Pull request merged**